### PR TITLE
Breaking blocks after matches

### LINKED ISSUE #607: Allow players to instantly break blocks once the game ends
**Describe the feature you'd like**
Once a match ends, players who participated in the game can instantly break all of the blocks on the map.

**Usefulness of the feature**
1000° fist vs. Warzone map
(Mainly for entertainment reasons, also to be able to let players get out of places they block themselves into)

**Additional context**
Saw it on another server and it was very enjoyable so please add ty

### PR DESCRIPTION
**Pull request by MrScopes**
If enabled, users with a permission (tgm.post.break) will be set into survival mode and will be able to break any block. tqnk helped a lot with this pr

Resolves #607

### CODE REVIEW

**Comment by MrScopes** [comment]
i meant to push to a new branch, but I'll test the new changes when I get home

**Comment by MrScopes** [comment]
finished

File: TGM/src/main/resources/config.yml [hunk 1]
```diff
@@ -52,3 +52,6 @@ server:
   # Text displayed after the match time on the tablist header.
   # Default: &f&lWARZONE
   tablist-name: "&f&lWARZONE"
+  # If players will be placed in to creative after the game is ended
+  # Any player is creative mode may break blocks during post game
```
Comment (BennyDoesTheStuff) [inline TGM/src/main/resources/config.yml]: Bit of a grammar mistake here ~ "Any player is creative mode"

Comment (BennyDoesTheStuff) [reply TGM/src/main/resources/config.yml]: You should probably also mention the permission to allow people to do this.

**Comment by jorgeberrex** [comment]
Two things that need to be changed:
- The config option should go under the `map` configuration section
- Instead of setting the gamemode to creative, it'd be better to set it to survival and set the BlockDamageEvent to instabreak

**Comment by MrScopes** [comment]
> Two things that need to be changed:
> 
> * The config option should go under the `map` configuration section
> * Instead of setting the gamemode to creative, it'd be better to set it to survival and set the BlockDamageEvent to instabreak

done

**Comment by MrScopes** [comment]
will this be merged?

File: TGM/src/main/java/network/warzone/tgm/modules/MatchResultModule.java [hunk 2]
```diff
@@ -43,7 +43,11 @@ public void onMatchResult(MatchResultEvent event) {
             if (spectators.containsPlayer(player)) {
                 player.playSound(location, Sound.ENTITY_WITHER_DEATH, 1000, 1);
             } else {
-                player.setGameMode(GameMode.ADVENTURE);
+                if (TGM.get().getConfig().getConfigurationSection("map").getBoolean("post-block-break") && player.hasPermission("tgm.post.break")) {
```
Comment (jorgeberrex) [inline TGM/src/main/java/network/warzone/tgm/modules/MatchResultModule.java]: `TGM.get().getConfig().getConfigurationSection("map").getBoolean("post-block-break")`
Can be simplified and provide a default value with
`TGM.get().getConfig().getBoolean("map.post-block-break", false)`

Comment (MrScopes) [reply TGM/src/main/java/network/warzone/tgm/modules/MatchResultModule.java]: only going off what i saw in other tgm code

**Pull request merged**